doc/source/installation.rst

```diff
@@ -63,23 +65,20 @@ For Ubuntu, run the following commands:
 .. code-block:: bash
 
   sudo apt-get update
-  sudo apt-get install -y cmake pkg-config build-essential autoconf curl libtool unzip flex bison psmisc python # we install python here because python2 is required to build the webui
+  sudo apt-get install -y build-essential curl unzip psmisc python # we install python here because python2 is required to build the webui
 
   # If you are not using Anaconda, you need the following.
   sudo apt-get install python-dev  # For Python 2.
   sudo apt-get install python3-dev  # For Python 3.
 
-  # If you are on Ubuntu 14.04, you need the following.
-  pip install cmake
-
   pip install cython==0.29.0
```

Comment:
ubuntu 14.04 needs "sudo" here for the system python, but it's probably ok to assume people can figure that out (the error message is good enough)